### DISCUSSION TITLE
How can we determine when a .m3u link has been last updated?

### DISCUSSION THREAD

**Discussion by YesTrustMe** [iptv-org/community#309] [Q&A] [answered]
I'm finding many IPTV links, especially from the USA are "dead" whilst they were working a few weeks ago.

**Comment by BellezaEmporium** [comment, marked as answer]
The best way for now is to watch the auto-update commits from the iptv bot.

**Reply by YesTrustMe** [reply]
How do I do that?

**Reply by PopeyeTheSai10r** [reply]
> How do I do that?

https://github.com/iptv-org/iptv/commits/master
